apps/vr-tests-react-components/package.json

```diff
@@ -65,6 +65,7 @@
     "@fluentui/react-text": "*",
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29732&commitId=779e82bd1093488a530c92855f1107fa4f3987a5&env=prod&clientType=FLUENTUI)


  <details><summary><b>TimePicker compat</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TimePicker compat.Default.default.chromium.png|0| Added|
|TimePicker compat.Show seconds.default.chromium.png|0| Added|
|TimePicker compat.With 12-hour format.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>